script/build.mjs

```diff
@@ -0,0 +1,52 @@
+import assert from 'node:assert/strict'
```

Comment:
Could you add a shebang and make this script executable?

```suggestion
#!/usr/bin/env node
import assert from 'node:assert/strict'
```

Reply:
Why? What’s wrong with scripts!

Reply:
Nothing! I just like them having a shebang and executable flag, so they are properly executable on unix-like systems.